core-web/apps/dotcms-ui/src/app/portlets/dot-apps/dot-apps-list/dot-apps-list.component.scss

```diff
@@ -44,12 +48,15 @@
     grid-gap: $spacing-4;
     grid-template-columns: repeat(auto-fill, minmax(23.42rem, 1fr));
     padding: $spacing-4;
+    overflow: auto;
+    max-height: 100%;
+    padding-bottom: 6.5rem;
```

Comment:
Why do we need this? this is the kind of CSS red flags we should keep an eye on.

Reply:
You right!
I changed the strategy of this